chore: run docs reviewer on all docs (#1982)
alchemyplatform/aa-sdk: docs/pages/authentication/login-methods/bring-your-own-auth.mdx

slug: wallets/authentication/login-methods/bring-your-own-auth
---

> Integrate your existing authentication provider and add smart wallet functionality to your app without changing your users' login experience. Smart Wallets supports JWT-based and OIDC-compliant authentication providers.

Reviewer: ```suggestion
Integrate your existing authentication provider and add smart wallet functionality to your app without changing your users' login experience. We support JWT-based and OIDC-compliant authentication providers.
```

## Next Steps

> ### Sending Transactions

Reviewer: ```suggestion
### Sending transactions
```

Once your users authenticate, start [sending transactions](/wallets/transactions/send/send-user-operations).

> ### Sponsor Gas for Transactions

Reviewer: ```suggestion
### Sponsor Gas for transactions
```
